SPECS/protobuf-c.spec

```diff
@@ -1,8 +1,8 @@
 # XCP-ng: imported from Fedora 30
 
 Name:           protobuf-c
-Version:        1.3.1
-Release:        2.3%{?dist}
+Version:        1.4.1
+Release:        1.0%{?dist}
```

Comment:
I don't think the release needs to end with 0, does it?

Reply:
Set to `1`